core/src/main/java/com/scalar/db/api/TransactionManagerCrudOperable.java

```diff
@@ -39,6 +39,18 @@ Optional<Result> get(Get get)
   List<Result> scan(Scan scan)
       throws CrudConflictException, CrudException, UnknownTransactionStatusException;
 
+  /**
+   * {@inheritDoc}
+   *
+   * @throws CrudConflictException if the transaction CRUD operation fails due to transient faults
+   *     (e.g., a conflict error). You can retry the transaction from the beginning
+   * @throws CrudException if the transaction CRUD operation fails due to transient or nontransient
+   *     faults. You can try retrying the transaction from the beginning, but the transaction may
+   *     still fail if the cause is nontranient
+   */
+  @Override
+  Scanner getScanner(Scan scan) throws CrudConflictException, CrudException;
```

Comment:
Also, added the `getScanner()` method and the `Scanner` interface to `TransactionManagerCrudOperable`.